Zappy_IA_Report/NN/Code/Tutorial7.py

```
@@ -0,0 +1,18 @@
+#python version 3.8
+from keras import *
```

Ruff (0.11.9)

`from keras import *` used; unable to detect undefined names

(F403)